scripts/mixins/randomthings.zs

```diff
@@ -0,0 +1,63 @@
+#loader mixin
+
+#mixin Mixin
+#{targets: "lumien.randomthings.entitys.EntityGoldenChicken"}
+zenClass MixinEntityGoldenChicken {
+    #mixin ModifyConstant
+    #{
+    #    method: "func_70636_d",
+    #    constant: {expandZeroConditions: GREATER_THAN_ZERO, ordinal: 1}
+    #}
+    function modifyIngotDropTimerCheck(value as int) as int {
+        return 7;
+    }
+
+    #mixin ModifyConstant
+    #{
+    #    method: "func_70636_d",
+    #    constant: {intValue: 3}
+    #}
+    function modifyItemDropCount(value as int) as int {
```

Comment:
In game, chickens consuming default Ore form of gold (instead of `Gold Native Cluster`) and produce default 3 pieces (instead of 7 pieces, one piece a tick)

![](https://i.imgur.com/qwWj3Ox.png)